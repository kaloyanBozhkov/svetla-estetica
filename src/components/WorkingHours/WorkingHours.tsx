import styles from './workingHours.module.scss'

const hours = [
        ['Lunedì', '09.00 - 20.00'],
        ['Martedì', '09.00 - 20.00'],
        ['Mercoledì', '09.00 - 20.00'],
        ['Giovedì', '09.00 - 20.00'],
        ['Venerdì', '09.00 - 20.00'],
        ['Sabato', '09.00 - 18.00'],
    ],
    WorkingHours = (): JSX.Element => (
        <div className={styles.workingHours}>
            {hours.map(([day, hour]) => (
                <p key={day}>
                    <span>{day}</span>
                    <span>{hour}</span>
                </p>
            ))}
        </div>
    )

export default WorkingHours
