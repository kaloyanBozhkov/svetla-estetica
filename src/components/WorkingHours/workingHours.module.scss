@import "scss/mixins.scss";
@import "scss/variables.scss";

.workingHours {
    width: 100%;
    display: grid;
    grid-template-columns: 1fr 1fr;
    height: 25.4rem;
    column-gap: 24rem;

    @include sidePaddings;
    padding-top: 4rem;
    padding-bottom: 4rem;

    > p {
        display: flex;
        justify-content: space-between;
        align-items: center;
        font-size: 2rem;
        line-height: 2.2rem;
        font-weight: lighter;
    }

    position: relative;
    &::before {
        content: '';
        width: 1px;
        position: absolute;
        left: 50%;
        transform: translateX(-50%);
        background-color: $lighterGrey;
        top: 20%;
        bottom: 20%;
    }

    @include breakpoint(tablet-portrait) {
        grid-template-columns: 1fr;
        height: 55rem;
        padding-top: 5rem;
        padding-bottom: 5rem;
    
        > p {
            font-size: 3rem;
        }

        &::before {
            display: none;
        }
    }
}
