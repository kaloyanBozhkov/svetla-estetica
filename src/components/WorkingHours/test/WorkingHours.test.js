import { shallow } from 'enzyme'
import React from 'react'
import WorkingHours from '../WorkingHours'

describe('testing WorkingHours', () => {
    it('should render component', () => {
        expect(shallow(<WorkingHours />)).toMatchSnapshot()
    })
})