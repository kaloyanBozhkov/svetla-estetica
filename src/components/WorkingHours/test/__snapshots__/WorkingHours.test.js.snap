// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing WorkingHours should render component 1`] = `
<div
  className="workingHours"
>
  <p
    key="Lunedì"
  >
    <span>
      Lunedì
    </span>
    <span>
      09.00 - 20.00
    </span>
  </p>
  <p
    key="Martedì"
  >
    <span>
      Martedì
    </span>
    <span>
      09.00 - 20.00
    </span>
  </p>
  <p
    key="Mercoledì"
  >
    <span>
      Mercoledì
    </span>
    <span>
      09.00 - 20.00
    </span>
  </p>
  <p
    key="Giovedì"
  >
    <span>
      Giovedì
    </span>
    <span>
      09.00 - 20.00
    </span>
  </p>
  <p
    key="Venerdì"
  >
    <span>
      Venerdì
    </span>
    <span>
      09.00 - 20.00
    </span>
  </p>
  <p
    key="Sabato"
  >
    <span>
      Sabato
    </span>
    <span>
      09.00 - 18.00
    </span>
  </p>
</div>
`;
